Content.Client/GameObjects/EntitySystems/StationEvents/ClientRadiationSystem.cs

```diff
@@ -0,0 +1,206 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Content.Client.GameObjects.Components.StationEvents;
+using JetBrains.Annotations;
+using Robust.Client.Graphics.Drawing;
+using Robust.Client.Graphics.Overlays;
+using Robust.Client.Interfaces.Graphics.ClientEye;
+using Robust.Client.Interfaces.Graphics.Overlays;
+using Robust.Client.Player;
+using Robust.Shared.GameObjects;
+using Robust.Shared.GameObjects.Systems;
+using Robust.Shared.Interfaces.GameObjects;
+using Robust.Shared.Interfaces.Map;
+using Robust.Shared.Interfaces.Timing;
+using Robust.Shared.IoC;
+using Color = Robust.Shared.Maths.Color;
+
+namespace Content.Client.GameObjects.EntitySystems.StationEvents
+{
+    /// <summary>
+    /// Handles showing the RadiationPulse event entities on the client side (using an overlay)
+    /// </summary>
+    [UsedImplicitly]
+    public sealed class ClientRadiationSystem : EntitySystem
```

Comment:
I refactored the OverlayEffectsComponent pretty much for this use-case. Just set the overlay via. the ServerOverlayEffectsComponent, and send a TimedOverlayParameter along so you can adjust the time on the overlay.